scripts/install_third_party.py

```diff
@@ -338,10 +339,74 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_redis_cli():
+    """This installs the redis-cli to the local oppia third_party directory so
+    that development servers and backend tests can make use of a local redis
+    cache. Redis-cli installed here (redis-cli-6.0.6) is different from the
+    redis package installed in manifest.json (redis-3.5.3). The redis-3.5.3
+    package detailed in manifest.json is the Python library that allows users to
+    communicate with any Redis cache using Python. The redis-cli-6.0.6 package
+    installed in this function contains C++ scripts for the redis-cli and
+    redis-server programs detailed below.
+
+    The redis-cli program is the command line interface that serves up an
+    interpreter that allows users to connect to a redis database cache and
+    query the cache using the Redis CLI API. It also contains functionality to
+    shutdown the redis server. We need to install redis-cli separately from the
+    default installation of backend libraries since it is a system program and
+    we need to build the program files after the library is untarred.
+
+    The redis-server starts a Redis database on the local machine that can be
+    queried using either the Python redis library or the redis-cli interpreter.
+    """
+    try:
+        # Pipe output to /dev/null for silence in console.
+        null = python_utils.open_file('/dev/null', 'w')
```

Comment:
It's also a bit unclear how this relates to the subprocess.call() below. Instead of doing this, maybe specify the stdout param in subprocess.call? (There should be examples online.)